src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java

```diff
@@ -146,6 +150,23 @@ public void testRectangleWithLineWidth10() throws IOException, URISyntaxExceptio
         assertEquals(10, imageComparison.getRectangleLineWidth());
     }
 
+    @Test
+    public void testShouldReturnARectangleList() throws IOException, URISyntaxException {
+        //Arrange
```

Comment:
Please, follow project style of the description the test cases,
e.g. `given`, `when`, `then`, `and` blocks.